app/views/_logo.html.erb

```diff
@@ -1,4 +1,3 @@
 <a id="logo" class="navbar-brand" href="<%= hyrax.root_path %>" data-no-turbolink="true">
-  <span class="glyphicon glyphicon-globe" role="img" aria-label="<%= application_name %>" aria-hidden="true"></span>
```

Comment:
Glyphicons have been dropped (they used to be a part of Bootstrap) but we do also support FontAwesome (4, i assume) if you want to preserve something similar; see #5292.

**Edit:** I see further down that you did in fact do that 😅.

Reply:
Yea, maybe I'm making a style decision but seems the Globe icon just gets in the way (to me).  Do we want to keep it?

Reply:
I think the “intent” of the globe is to be a sort of standin for organization logos; see for example:

+ http://libarchive.linnbenton.edu
+ https://scholarspace.library.gwu.edu

But I’ve also found ones which just deleted it:

+ https://archive.kingsfund.org.uk

Personally, I don’t find having a hardcoded globe that institutions would have to manually replace to be particularly helpful, and am fine with just getting rid of it.